packages/sdk/.size-limit.ts

```diff
@@ -0,0 +1,10 @@
+import type { SizeLimitConfig } from 'size-limit';
+
+module.exports = [
+    {
+        name: "GRPC web client tree shaking",
+        path: "lib/esm/index.js",
+        limit: "100 KB",
```

Comment:
I think we should set the limit, just above the current size (53.08 kB), such that we catch if the bundle size grows.
So maybe put `54 kB` instead